2 - RTB/Documentazione esterna/Piano di Progetto/Piano di Progetto.typ

```diff
@@ -1473,83 +2066,564 @@ Diversi ruoli risultano avere un monte ore minore rispetto a quello preventivato
+#let data = (
+  ("Responsabile", 2, 2),
+  ("Amministratore", 2, 2),
+  ("Analista", 2, 2),
+  ("Progettista", 0, 1),
+  ("Programmatore", 4, 4),
+  ("Verificatore", 8, 8),
+)
+#let x-coordinates = compute-labels-x-coordinate(data, role-chart-size)
+#let y-coordinates = compute-labels-y-coordinate(data, role-chart-size)
+
+#figure({
+  import draw: *
+  canvas({
+    chart.barchart(..barchart-config, data)
+    let i = 0
+    while(i < data.len()) {
+      content(
+        (x-coordinates.at(i).at(0), y-coordinates.at(i).at(0)),
+        [#data.at(i).at(1)],
+        ..barchart-label-config
+      )
+      content(
+        (x-coordinates.at(i).at(1), y-coordinates.at(i).at(1)),
+        [#data.at(i).at(2)],
+        ..barchart-label-config
+      )
+      i += 1
+    }
+  })},
+  caption: "Suddivisione oraria per ruolo, consuntivo sprint 10",
+  kind: "chart",
+  supplement: "Grafico"
+)
+
+La presenza del Progettista, non pianificata, deriva dalla necessità di decidere quale software adottare (tra Next.js ed Express.js) per l'implementazione di un componente del PoC. Entrambe le tecnologie sono state approfondite ed implementate durante lo sviluppo dei PoC, ma non era ancora stata presa una scelta definitiva. Inoltre, la redazione della presentazione per il colloquio RTB ha nuovamente richiesto l'intervento del Progettista.
+
+=== Monitoraggio costi e ore
+
+#figure(
+  table(
+    columns: 3,
+    [*Ruolo*], [*Ore rimanenti*], [*Budget rimanente*],
+    [Responsabile],     [43],     [€ 1290],
+    [Amministratore],     [42],     [€ 840],
+    [Analista],     [51],     [€ 1275],
+    [Progettista],     [29],     [€ 725],
+    [Programmatore],     [165],     [€ 2475],
+    [Verificatore],     [105],     [€ 1575],
+    [Rimanente],     [435],     [€ 8180],
+  ),
+  caption: "Monitoraggio sprint 10"
+)
+
+== Sprint 11 dal 14-01-2024 al 21-01-2024
+
+=== Obiettivi raggiunti
+- Inviata la candidatura al Prof. Cardin per il colloquio RTB;
+- Al Piano di Progetto sono state apportate modifiche minori per migliorare la leggibilità dei dati;
+- Il sito web è accessibile in ogni sua parte e comprende i riferimenti di ogni documento prodotto dal gruppo;
+- I contenuti della presentazione per il colloquio RTB sono stati definiti e il documento è stato completato.
+
+=== Obiettivi mancati
+Nessuno.
+
+=== Problematiche
+Durante il meeting di retrospettiva sono sorte le seguenti problematiche:
+- carico di lavoro asimmetrico tra i membri del gruppo a causa dei differenti esami da sostenere.
+
+=== Risoluzioni attuate
+Nessuna. La situazione di carico di lavoro asimmetrico è destinata a migliorare con l'approcciarsi della fine della sessione d'esami invernale.
+
+=== Panoramica dei costi effettivi
+
+#figure(
+  table(
+    columns: 8,
+    [*Membro*], [*Responsabile*], [*Amministratore*], [*Analista*], [*Progettista*], [*Programmatore*], [*Verificatore*], [*Totale*],
+    [Banzato],     [0],     [0],     [1 (-1)],     [0],     [0],     [0],     [1 (-1)],
```

Comment:
alcuni dati della tabella non sono in linea con quelli del preventivo (per esempio Banzato ha segnato come Analista "1(-1)" però nella tabella di preventivo sprint 11 è segnato 4 ore.)

Reply:
Hai ragione, ma purtroppo il documento attualmente in `main` non è corretto e ha diversi errori. Fa fede il foglio elettronico, infatti se vedi il preventivo dello sprint 11 in questo documento torna tutto